### ISSUE TITLE
Incorrect flagging of missing aria-label attributes

### ISSUE THREAD

**Issue by galbaras** [janreges/siteone-crawler#112]
SiteOne flags hidden fields and fields with labels as missing `aria-label` attributes, which is incorrect.

Specifically for labels, this seems to be happening when the field is outside the label, and they are linked via the `for` attribute. For example:

```
<div class="woocommerce">
	<form id="searchform" action="https://www.plastic2go.com.au" method="get">
		<label class="screen-reader-text" for="woocommerce-product-search-field-0">Find products</label>
		<input id="woocommerce-product-search-field-0" name="s" type="text" value="" placeholder="Find products"> 
		<input id="searchsubmit" class="button" type="submit" value="Find Now"> 
		<input name="post_type" type="hidden" value="product">
	</form>
</div>
```
